nucliadb_node/src/bin/writer.rs

```diff
@@ -108,25 +108,11 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         }
     });
 
-    let report = NodeReport::new(host_key.to_string())?;
-
     let metrics_task = tokio::spawn(async move {
```

Comment:
At some point it would be better to add this as static functions on the file instead of closures inside a huge main.

Reply:
I agree.

Reply:
Could be part of another PR, isn't it ?

Reply:
yes, should be part of another PR